levels/main.gd

```diff
@@ -1,14 +1,30 @@
 extends Node2D
 
+#const EnemySpawner = preload("res://characters/enemies/enemy_spawner.gd")
 
-const SKELETON = preload("res://characters/enemies/skeleton.tscn")
-var spawn_location := Vector2(-100, - 150)
 
-
-func _init() -> void:
-	for i in range(10):
-		var skeleton_instance : Skeleton = SKELETON.instantiate()
-		skeleton_instance.global_position = spawn_location
-		add_child(skeleton_instance)
+#func _init() -> void:
+	#for i in range(10):
+		#var skeleton_instance : Skeleton = SKELETON.instantiate()
+		#skeleton_instance.global_position = spawn_location
+		#add_child(skeleton_instance)
+		#
+		#spawn_location.x += 40
 		
-		spawn_location.x += 40
+
+func spawn_single_mob()-> void:
+	#more mobs-possibilyty for randomness
+	var new_mob = preload("res://characters/enemies/skeleton.tscn").instantiate()
```

Comment:
preload should be probably moved to the class scope as a constant and then it can be instantiated inside both functions